src/reducers/versions.spec.tsx

```diff
@@ -352,6 +352,39 @@ describe(__filename, () => {
       expect(_log.debug).toHaveBeenCalled();
       expect(versionsState).toEqual(previousState);
     });
+
+    it('logs a debug message when headVersion is missing on loadDiff()', () => {
+      const addonId = 1;
+      const baseVersionId = 2;
+      const path = 'some/other/file.js';
+      const _log = createFakeLogger();
+
+      const version = {
+        ...fakeVersionWithDiff,
+        file: {
+          ...fakeVersionWithDiff.file,
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          selected_file: path,
```

Comment:
Why do you need to override `selected_file` with anything? It doesn't seem to have any impact on what you are testing here. It only cares about whether the version being requested is present or not.

Reply:
It does not matter indeed.